README.md

```diff
@@ -1,14 +1,5 @@
 # Alteryx Open Source Demos 
 
```

Comment:
<img width="1078" alt="Screen Shot 2021-12-14 at 3 10 09 PM" src="https://user-images.githubusercontent.com/8726321/146072158-9f72d912-a066-400e-990e-f3ae606f1aad.png">

We could use this image.

Reply:
@gsheni Is that image hosted somewhere already that we can pull from, or do we need to add it?

Reply:
Added
https://woodwork-web-images.s3.amazonaws.com/alteryx-oss-libraries.png

Reply:
We should center the image

Reply:
@gsheni Updated README.md to use this image and added the `Built at Alteryx Innovation Labs` footer like we have in other repos.